examples/dt/uni05epsilon/control-plane/networking/nncp/kustomization.yaml

```
@@ -21,3 +21,117 @@ components:
 
 resources:
   - values.yaml
+
+patches:
+  - target:
+      kind: NodeNetworkConfigurationPolicy
+      name: master-0
+    patch: |-
+      - op: add
+        path: /spec/desiredState/interfaces/-
+        value:
+          description: storagemgmt vlan host interface
+          name: storagemgmt
+          state: up
+          type: vlan
+          mtu: 1500
+          vlan:
+            base-iface: _replaced_
+            id: _replaced_
+
+  - target:
+      kind: NodeNetworkConfigurationPolicy
+      name: master-1
+    patch: |-
+      - op: add
+        path: /spec/desiredState/interfaces/-
+        value:
+          description: storagemgmt vlan host interface
+          name: storagemgmt
+          state: up
+          type: vlan
+          vlan:
+            base-iface: _replaced_
+            id: _replaced_
+
+  - target:
+      kind: NodeNetworkConfigurationPolicy
+      name: master-2
+    patch: |-
+      - op: add
+        path: /spec/desiredState/interfaces/-
+        value:
+          description: storagemgmt vlan host interface
+          name: storagemgmt
+          state: up
+          type: vlan
+          vlan:
+            base-iface: _replaced_
+            id: _replaced_
+
+replacements:
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.storagemgmt.base_iface
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+          name: master-0
+        fieldPaths:
+          - spec.desiredState.interfaces.[name=storagemgmt].vlan.base-iface
+
+  - source:
+      kind: ConfigMap
+      name: network-values
+      fieldPath: data.storagemgmt.vlan
+    targets:
+      - select:
+          kind: NodeNetworkConfigurationPolicy
+          name: master-0
+        fieldPaths:
+          - spec.desiredState.interfaces.[name=storagemgmt].vlan.id
```

You can simplify this and remove the need for separate ones for master-1 and master-2, since the VLAN ID is the same across all nodes.
```suggestion
  - source:
      kind: ConfigMap
      name: network-values
      fieldPath: data.storagemgmt.vlan
    targets:
      - select:
          kind: NodeNetworkConfigurationPolicy
          name: .*
        fieldPaths:
          - spec.desiredState.interfaces.[name=storagemgmt].vlan.id
```